example/exampleUser/pluginSpec.js

```diff
@@ -26,7 +26,7 @@ import {
 } from '../../src/utils/testing';
 import ExampleUserProvider from './ExampleUserProvider';
 
-describe("The Example User Plugin", () => {
+xdescribe("The Example User Plugin", () => {
```

Comment:
This test leaks login forms into the display:
<img width="359" alt="image" src="https://user-images.githubusercontent.com/9853862/154983206-ea2f9138-8837-442d-9534-653cada10e19.png">

causing plugin tests that assume absolute positioning to fail, e.g., the Menu API:
```
 Menu Placements
      ✗ default menu position BOTTOM_RIGHT (5ms)
	Error: Expected 141 to equal 16.
	    at <Jasmine>
	    at UserContext.eval (webpack-internal:///./src/api/menu/MenuAPISpec.js:198:25)
	    at <Jasmine>

      ✗ menu position BOTTOM_RIGHT (5ms)
	Error: Expected 141 to equal 16.
	    at <Jasmine>
	    at UserContext.eval (webpack-internal:///./src/api/menu/MenuAPISpec.js:214:25)
	    at <Jasmine>
```
This is probably because the form never cleans itself up:
https://github.com/nasa/openmct/blob/master/example/exampleUser/ExampleUserProvider.js#L99
It leaked into the code base broken as the regex (fixed in this PR) was excluding the `example` directory, and thus never running the test.

Reply:
If you think that the leaky test can get fixed in a single day, we just put it in this PR given that everybody else is out.

Can you think of a e2e scenario to cover this?

Reply:
I can’t think of one off the top of my head, and I’m out the rest of the day, so I guess I should file a ticket!

Reply:
https://github.com/nasa/openmct/issues/4886